procedural/src/lib.rs

```diff
@@ -71,58 +71,32 @@ pub fn openzeppelin_construct_runtime(_: TokenStream, tokens: TokenStream) -> To
 }
 
 /// Runtime API macro wrapper
-/// This macro wraps the `impl_runtime_api` macro and provides our implementations of them.
+/// This macro wraps the `impl_runtime_api` macro and provides a simpler interface.
 /// It also works in the same groupings as our abstractions pallets and to get the runtime API implementations you only need to provide some types.
```

Comment:
@KitHat I wasn't able to understand this phrase. Can you try to make it a bit clearer?

Reply:
I meant that this macro provides our implementations of runtime apis